terraform.gpaas-azure-migration/container-apps-hosting.tf

```diff
@@ -15,7 +15,6 @@ module "azure_container_apps_hosting" {
   container_secret_environment_variables = local.container_secret_environment_variables
 
   enable_mssql_database = local.enable_mssql_database
-  enable_redis_cache    = local.enable_redis_cache
```

Comment:
We might have to leave this parameter here for the Azure stuff - @Stretch96 can you confirm?

Reply:
I _think_ it should be fine without. From the docs, here: https://github.com/DFE-Digital/terraform-azurerm-container-apps-hosting

The default is false, and it's not a mandatory field (no in the last column)

enable_redis_cache | Set to true to create an Azure Redis Cache, with a private endpoint within the virtual network | bool | false | no
-- | -- | -- | -- | --

@Stretch96 - does that sound about right?